### PR TITLE
MS. Hamilton 15 - new description

### PR DESCRIPTION
**Pull request by holfordm**

### CODE REVIEW

File: places.xml [hunk 1]
```diff
@@ -592,6 +592,10 @@
                 <!--<place type="settlement" xml:id="place_7028492">
                     <placeName  type="index" source="TGN">Ketteringham</placeName></place> -->
                 
+                <place type="settlement" xml:id="place_7083534">
+                    <placeName>Wellmitz</placeName>
```
Comment (adunning) [inline places.xml]: <https://www.geonames.org/2811453/>

File: collections/Hamilton/MS_Hamilton_15.xml [hunk 2]
```diff
@@ -49,37 +47,267 @@
...
                   <msItem xml:id="MS_Hamilton_15-item1" n="1">
-                     <author key="person_4472713">
+                     <locus>(fols. 1ra-238va; 238vb blank)</locus>
+                     <author key="person_84969355">Conrad de Brundelsheim ('Soccus')</author>
+                     <title key="work_1309">Sermones de tempore</title>
+                     
+                     <note>(pars hyemalis, with supplementary Lenten sermons)</note>
+                     <incipit><cit><quote>Exsurge amortuis et illuminabit te christus</quote> <bibl>Ephe. v<hi rend="superscript">o</hi>.</bibl></cit> Cogitanti michi quod dominum patrem peccatis meis grauiter offenderim</incipit>
+                    
+                     <msItem><note>(fols. 1r-58r): nos. 1-36.</note></msItem>
```
Comment (adunning) [inline collections/Hamilton/MS_Hamilton_15.xml]: Clarify that these numbers are from Schneyer?

File: collections/Hamilton/MS_Hamilton_15.xml [hunk 3]
```diff
@@ -49,37 +47,267 @@
...
+                        <note>Cf. Schneyer VIII, p. 574, no. 33.</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 165ra-165va)</locus>
+                        <incipit><quote>Ecce nunc tempus acceptabile</quote> Nota super illo verbo Gal. ijo, In faciem restiti ei</incipit>
+                        <explicit>diuersitas intelligencie, quam nobis dominus</explicit>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 165va-169ra)</locus>
+                        <author key="person_89204668">Bertrandus de Turre</author> <title key="work_6170">Sermones de tempore et quadragesimales</title> <note>(nos. 105-106; Schneyer I, pp. 513-14)</note>
```
Comment (adunning) [inline collections/Hamilton/MS_Hamilton_15.xml]: Is it a problem to have works from other authors nested under an `<msItem>` that gives Conrad as the author of the whole?

File: collections/Hamilton/MS_Hamilton_15.xml [hunk 4]
```diff
@@ -49,37 +47,267 @@
...
+                     <incipit>De modo obseruandi interdictum memoriale quoddam simplicibus colligendo</incipit>
+                     <explicit defective="true">non diffiniendo sed propo</explicit>
+                     <note>Ends imperfect (at f. 11r of ISTC ia00629000).</note>
+                     <!-- https://app.digitale-sammlungen.de/bookshelf/bsb00044163/view?currentUrl=%2Fview%2Fbsb00044163%3Fview%3DImageView&manifest=https%3A%2F%2Fapi.digitale-sammlungen.de%2Fiiif%2Fpresentation%2Fv2%2Fbsb00044163%2Fmanifest&canvas=https%3A%2F%2Fapi.digitale-sammlungen.de%2Fiiif%2Fpresentation%2Fv2%2Fbsb00044163%2Fcanvas%2F15&view=ImageView -->
+                  </msItem>
+                  <msItem>
+                     <locus>(fols. 245ra-246va; 246vb, 247r blank)</locus>
+                     <title type="desc" key="work_15091">Treatise on the mass</title>
+                     <incipit defective="true">in f<hi rend="superscript">r</hi>mis. Nam de miraculo veteris panis exodi</incipit>
+                     <explicit>omnem suam dulcedinem infundit in epulum quod nobis prestare dignetur</explicit>
```
Comment (adunning) [inline collections/Hamilton/MS_Hamilton_15.xml]: Probably a variant of Conrad de Brundelsheim, sermon no. 124.

File: collections/Hamilton/MS_Hamilton_15.xml (hunk 3 above)
Comment (holfordm) [reply collections/Hamilton/MS_Hamilton_15.xml]: Possibly for some kinds of processing? Not for anything that we do at the moment on the website. It does I suppose indicate that these items circulated as Conrad.

File: collections/Hamilton/MS_Hamilton_15.xml (hunk 4 above)
Comment (holfordm) [reply collections/Hamilton/MS_Hamilton_15.xml]: Good thought but it doesn't seem to be : 1484 edition http://www.ub.uni-koeln.de/cdm/compoundobject/collection/inkunabeln/id/109982/rec/1 around image 436

**Pull request merged**